en/docs/apis/restapis/user-store.yaml

```diff
@@ -129,10 +175,12 @@ paths:
       operationId: getUserStoreByDomainId
       description: >
         This API provides the capability to retrieve the configurations of
-        secondary user store based on its domain id.
+        secondary user store based on its domain id.<br>
```

Comment:
```suggestion
        a secondary user store based on its domain id.<br>
```

Reply:
Does this API retrieve data about a single user store or multiple? lets update the article accordingly.